src/SDF_TEST.cc

```diff
@@ -538,6 +538,47 @@ TEST(SDF, Version)
   EXPECT_STREQ(SDF_VERSION, sdf::SDF::Version().c_str());
 }
 
+/////////////////////////////////////////////////
+TEST(SDF, EmbeddedSpec)
+{
+  std::string result;
+  std::string output;
+
+  result = sdf::SDF::EmbeddedSpec("actor.sdf", false);
+  EXPECT_NE(result.find("<!-- Actor -->"), std::string::npos);
+  EXPECT_NE(result.find("<element name=\"actor\" required=\"*\">"), std::string::npos);
+  result = sdf::SDF::EmbeddedSpec("actor.sdf", true);
+  EXPECT_NE(result.find("<!-- Actor -->"), std::string::npos);
+  EXPECT_NE(result.find("<element name=\"actor\" required=\"*\">"), std::string::npos);
+
+  result = sdf::SDF::EmbeddedSpec("root.sdf", false);
+  EXPECT_NE(result.find("SDFormat base element"), std::string::npos);
+  EXPECT_NE(result.find("name=\"version\" type=\"string\""), std::string::npos);
+  result = sdf::SDF::EmbeddedSpec("root.sdf", true);
+  EXPECT_NE(result.find("SDFormat base element"), std::string::npos);
+  EXPECT_NE(result.find("name=\"version\" type=\"string\""), std::string::npos);
+}
+
+TEST(SDF, EmbeddedSpecNonExistent)
+{
+  std::string result;
+  std::string output;
+
+  testing::internal::CaptureStderr();
+  result = sdf::SDF::EmbeddedSpec("unavailable.sdf", false);
+  output = testing::internal::GetCapturedStderr();
+  EXPECT_NE(output.find("Unable to find SDF filename"), std::string::npos);
```

Comment:
if you replace the `;` with ` << output;`, it should print the entire output if the expectation fails. This may help with debugging the failure on windows

Reply:
Looks like there is an issue with the CI now: https://build.osrfoundation.org/job/sdformat-ci-pr_any-windows7-amd64/3327/

Reply:
Your suggestion didn't helped much - nothing was printed, see https://build.osrfoundation.org/job/sdformat-ci-pr_any-windows7-amd64/3328/
However, I found a similar approach in parser_TEST.cc that doesn't depend on GTest and which hopefully will also work here. Otherwise I'll comment this test out for Windows and someone else has to fix it later with direct access to a Windows machine...